### ISSUE TITLE
Support

### ISSUE THREAD

**Issue by AScripnic** [congajs/conga-annotations#9]
Hello,

I'd like to ask if this repo is maintained, if there is any chances a PR would be reviewed and merged if passes the review.
Sorry for creating an issue for this.

**Comment by createvibe** [comment]
Hello!  Yes we are still around :) If you have a PR you want merged, we will review it.  What feature or issue are you trying to expand on?

**Comment by AScripnic** [comment]
@createvibe I want to add the annotation parsing for async functions:
```javascript
class MyClass {
  /**
   * My annotation
   * @path path
   * @param {*} req
   * ....
   */
  async test() {
    await this.asyncFunction();
  }
}

```
Thank you

**Comment by AScripnic** [comment]
@createvibe Could you check if it's okay:
https://github.com/congajs/conga-annotations/pull/10

**Comment by createvibe** [comment]
looks good to me.  merged.  thank you!

**Comment by AScripnic** [comment]
Hello @createvibe
do you know when it'll be published?

**Comment by createvibe** [comment]
@AScripnic 
published + @conga/annotations@1.0.3 in npm registry
